syft/frameworks/torch/tensors/interpreters/precision.py

```diff
@@ -62,15 +62,9 @@ def fix_precision(self):
         rational = self.child
 
         upscaled = (rational * self.base ** self.precision_fractional).long()
-        field_element = upscaled % self.field
+        field_element = upscaled % self.field  # Should we raise a warning if upscaled > field?
```

Comment:
I think it makes a lot of sense to raise an warning here. I would argue that we should raise an warning if |upscaled| > field/2 since every data point that meets that condition will be embedded incorrectly. In the past we have considered raising an error if base**precision_fractional > field but this is a much more reasonable place to check